common/views/components/SegmentedControl/SegmentedControl.js

```diff
@@ -0,0 +1,162 @@
+        </button>
+        <div
+          id={id}
+          className={classNames({
+            [spacing({s: 3}, {padding: ['left', 'right']})]: true,
+            [spacing({s: 4}, {padding: ['top']})]: true,
+            'segmented-control__body': true,
+            'bg-white js-show-hide-drawer': true
+          })}>
+
+          <span className={classNames({
+            [spacing({s: 3}, {margin: ['bottom']})]: true,
+            'segmented-control__label': true,
+            'block': true
+          })}>See:</span>
+
+          <ul className='segmented-control__drawer-list js-segmented-control__drawer-list no-margin no-padding plain-list'>
+            {items.map((item, i) => (
+              <li
+                key={item.id}
+                className={classNames({
+                  [font({s: 'WB6'})]: true,
+                  [spacing({s: 3}, {padding: ['top', 'bottom']})]: true,
+                  'border-top-width-1': i === 0,
+                  'segmented-control__drawer-item': true,
+                  'border-bottom-width-1': true,
+                  'border-color-smoke': true
+                })}>
+                <a
+                  onClick={() => trackEvent({
+                    category: 'component',
+                    action: 'whats-on-daterange-picker:click',
+                    label: 'title:' + item.text
+                  })}
+                  data-track-event={JSON.stringify({
+                    category: 'component',
+                    action: 'whats-on-daterange-picker:click',
+                    label: 'title:' + item.text
+                  })}
+                  href={item.url}
+                  className={classNames({
+                    'js-segmented-control__drawer-link': true,
+                    'segmented-control__drawer-link': true,
+                    'block': true,
+                    'plain-link': true,
+                    'js-trap-reverse-start': i + 1 === items.length,
+                    'js-proxy-tablink': isTabControl
+                  })}>
+                  {item.text}
+                </a>
+              </li>
+            ))}
+          </ul>
+          <span className='visually-hidden js-trap-end'>reset focus</span>
+        </div>
+      </div>
+      <ul className={classNames({
+        'js-segmented-control__list': true,
+        'segmented-control__list': true,
+        'no-margin': true,
+        'no-padding': true,
+        'plain-list': true,
+        'border-width-1': true,
+        'border-color-black': true,
+        'rounded-diagonal': true,
+        'overflow-hidden': true,
+        'js-tablist segmented-control__list--inline': isTabControl
+      })}>
+        {items.map((item, i) => (
+          <li
+            key={item.id}
+            className={classNames({
+              [font({s: 'WB7'})]: true,
+              'border-right-width-1 border-right-color-black': i !== items.length - 1,
+              'segmented-control__item line-height-1 flex': true,
```

Comment:
how do you decide whether each string should contain only one class?

Reply:
I've been tending to favour 1 classname per property... thoughts?

Reply:
Yeah, I'm down with that, but looks like a few (e.g. here) have made it into multiple classes per string

Reply:
actually – I think it's only this line (the others make sense as they're instances of multiple classes being conditionally added based on some dynamic condition)